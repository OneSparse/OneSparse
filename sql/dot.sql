\setenv dot_source :dot_source
\! echo $dot_source | dot -Tsvg | sed '/<!DOCTYPE/,/>/d; /<?xml/,/>/d' | sed '1i<div>' | sed '$a</div>' | sed 's/^/-- /'
