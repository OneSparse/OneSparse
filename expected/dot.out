\setenv dot_source :dot_source
\! echo $dot_source | dot -Tsvg -Nnodesep=0.1 -Nranksep=0.1 -Nwidth=0.05 -Nheight=0.05 -Nfontsize=8 -Efontsize=8 | sed '/<!DOCTYPE/,/>/d; /<?xml/,/>/d' | sed '1i<div>' | sed '$a</div>\n' | sed 's/^/-- /'
Error: <stdin>: syntax error in line 1 near ':'
